builds/misc/packages.yaml

```diff
@@ -3,6 +3,7 @@ trigger:
   branches:
     include:
       - master
+      - iiot
```

Comment:
what if changes in `iiot` trigger a build that will not have up-to-date master changes? will it affect any other downstream pipelines/releases?

Reply:
iiot branch (any branch) should always be complie-able and it should work all the time, even it doesn't have up-to-date changes from master.